### PR TITLE
Added Looping Through Objects

### PR DESCRIPTION
**Pull request by KabirCampwala**
### Description

I've added looping through objects topic under JavaScript > Concepts > Loops.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by KabirCampwala** [comment]
Hey @Dusch4593, I don't want to disturb you but can you explain what is going on with this request now? 🤔

**Comment by Dusch4593** [comment]
> Hey @Dusch4593, I don't want to disturb you but can you explain what is going on with this request now? 🤔

**Comment by KabirCampwala** [comment]
> > Hey @Dusch4593, I don't want to disturb you but can you explain what is going on with this request now? 🤔
> 
> Hey @KabirCampwala sorry about the close/re-open with the PR; the mouse slipped 😆 We are currently trying to complete reviews for some of the previous PRs that came before this one. Once they are finished, we'll be ready to have someone take a look at this PR.

Alright! No problem.

File: content/javascript/concepts/loops/loops.md [hunk 1]
```diff
@@ -159,6 +159,24 @@ for (let i = 0; i < array.length; i++) {
...
+
+Using `Object.entries` an array of arrays is created which will contain both the property and the value respectively.
+
+```js
+const objectK = {
+  name: 'Codecademy',
+  age: 10
+};
+```
+The output would look like:
```
Comment (Dusch4593) [inline content/javascript/concepts/loops/loops.md:174]:
```suggestion

The output would look like:

```

File: content/javascript/concepts/loops/loops.md [hunk 2]
```diff
@@ -159,6 +159,24 @@ for (let i = 0; i < array.length; i++) {
...
+```
```
Comment (Dusch4593) [inline content/javascript/concepts/loops/loops.md:175]: To keep consistent with the rest of the output snippets in this entry, let's change this to ```shell

File: content/javascript/concepts/loops/loops.md (hunk 1 above)
Comment (Dusch4593) [reply content/javascript/concepts/loops/loops.md:174]: Remember to always keep one empty line on top of and below a code snippet. 😃

Comment (KabirCampwala) [reply content/javascript/concepts/loops/loops.md:174]: Okay!

File: content/javascript/concepts/loops/loops.md (hunk 2 above)
Comment (KabirCampwala) [reply content/javascript/concepts/loops/loops.md:175]: Alright!

File: content/javascript/concepts/loops/loops.md [hunk 3]
```diff
@@ -159,6 +159,24 @@ for (let i = 0; i < array.length; i++) {
...
+
+To loop through objects use `Object.entries` which will first convert the objects into an array and then loop through that array. 
+
+Using `Object.entries` an array of arrays is created which will contain both the property and the value respectively.
+
+```js
+const objectK = {
+  name: 'Codecademy',
+  age: 10
+};
```
Comment (Dusch4593) [inline content/javascript/concepts/loops/loops.md:172]: It seems we're missing the loop code that uses `Object.entries()`.

Comment (Dusch4593) [reply content/javascript/concepts/loops/loops.md:172]:
```js
for(const [key, value] of Object.entries(objectK)) {
  console.log(`${key}: ${value}`);
}
```

Comment (yangc95) [reply content/javascript/concepts/loops/loops.md:170]: @KabirCampwala, please refer to @Dusch4593's comment above as to what the js code block is missing!

I'd suggest to add `myEntries = Object.entries(objectK))` and `console.log(myEntries)` or just `console.log(Object.entries(objectK))` first with the output `[ [ 'name', 'Codecademy' ], [ 'age', 10 ] ]` commented out, then do the `for` loop suggested above:

```js
for(const [key, value] of Object.entries(objectK)) {
  console.log(`${key}: ${value}`);
};
```
Finally, we would add the output commented out on the following lines. This way, we won't need a separate block for the outputs.

In addition, to make these clear to the reader, we should add a line about what is happening before/after the code block. You can determine where you wish to place this explanation.

**Review by yangc95 — COMMENTED** [review]
@KabirCampwala, only one comment about the code example! Please tag me and/or @Dusch4593 once you've resolved that issue ☺️

**Review by yangc95 — APPROVED** [review]

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍 

Made a few last-minute commits prior to merging (mostly stylistic):

- aa416be3e3b40e5a43dd3acbfc5cd7f282a24bab
- f15ee0073b66b89148ae8e9a852d13ee2b91f8d0
- ab2c2cb70cd65a497a6a05eb97eee11ca3775204
- 83911901d8ca1ebef9e84232b0dcd58a60889be0

cc: @KabirCampwala @yangc95

**Comment by Dusch4593** [comment]
Sweet! Congrats, Kabir! Your changes are live in the entry: https://www.codecademy.com/resources/docs/javascript/loops

**Pull request merged**